### PR TITLE
Config update, account linking update (still not 100%) pull request

### PR DESCRIPTION
**Pull request by zachschultz**

### CODE REVIEW

File: src/fbbot/sendUtils.js [hunk 1]
```diff
@@ -110,18 +112,25 @@ function sendCourses(recipientId) {
           .then((succ) => {
             const courses = succ.data;
             courses.forEach((course) => {
-              console.log(course.name);
               if (!course.name) {
                 course.name = ' ';
               }
+              const courseURL = `https://ufl.instructure.com/courses/${course.id}`;
```
Comment (shqie) [inline src/fbbot/sendUtils.js]: @zachschultz  I thnk you hd it better before where `https://ufl.instructure.com/` was as variable in the config files?

File: README.md [hunk 2]
```diff
@@ -4,27 +4,40 @@ Messenger bots and API
 # Instructions
```
Comment (shqie) [inline README.md:4]: @zachschultz Would it be possible to have instructions when you want it to run it in DEV and when you want to run it in PRODUCTION? Or better when you run it locally on the computer is DEV. And when we deploy in PROD is Heroku?

File: README.md [hunk 3]
```diff
@@ -4,27 +4,40 @@ Messenger bots and API
...
+3. Common config, like Facebook app (not including `callbackURL`) info will go in `default.json`, config for your local development should go in `dev.json`, and config for production will
+go in `heroku.json`
+4. After changing your config, run `config-heroku save` to deploy it to Heroku
+* Alternatively, you can run `config-heroku hook add heroku HEROKU_CONFIG <YOUR APP NAME HERE>`
+to create a git hook that will auto-update your `HEROKU_CONFIG` file on Heroku when you make a commit
+6. To use config in your code, import it normally, like `import config from 'config-heroku'`, and access
+it like you would a normal config.json file, i.e. `config.PROPERTY_NAME`
 
-These files contain the same JSON5 structure:
+Our `default.json` looks like:
```
Comment (shqie) [inline README.md:26]: @zachschultz  it looks like you are explaining what should go in the `default.json` and `heroku.json`, but what should go in the `dev.json`

Comment (zachschultz) [reply README.md:26]: Just added dev.json example

File: README.md (hunk 2 above)
Comment (zachschultz) [reply README.md:4]: Added dev instructions for gulp

File: src/fbbot/sendUtils.js [hunk 4]
```diff
@@ -110,18 +112,25 @@ function sendCourses(recipientId) {
           .then((succ) => {
             const courses = succ.data;
             courses.forEach((course) => {
-              console.log(course.name);
               if (!course.name) {
                 course.name = ' ';
               }
+              const courseURL = `${CANVAS_API}courses/${course.id}`;
```
Comment (zachschultz) [inline src/fbbot/sendUtils.js:118]: Using canvas api here, must've been a mistake from before

**Pull request merged**